README.md

```diff
@@ -35,7 +35,8 @@ The zip archive will be found under `./kafka-connect-runtime/build/distributions
 | iceberg.control.commitThreads            | Number of threads to use for commits, default is (cores * 2)                                                  |
 | iceberg.catalog                          | Name of the catalog, default is `iceberg`                                                                     |
 | iceberg.catalog.*                        | Properties passed through to Iceberg catalog initialization                                                   |
-| iceberg.hadoop.*                         | Properties passed through to Hadoop configuration                                                             |
+| iceberg.hadoop-conf-dir                  | If specified, Hadoop config files in this directory will be loaded                                            |
```

Comment:
should this mention the order and the files that you have mentioned in the PR description?

Reply:
Yes, definitely. I added a section on Hadoop config.

Reply:
thanks